css/home.css

```diff
@@ -0,0 +1,774 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  overflow: hidden;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* top menu */
+.top-menu {
+  background: rgb(15, 15, 15);
+}
+
+.top-menu-ul {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 10%;
+}
+
+.top-menu-ul li {
+  padding: 1%;
+}
+
+.top-menu-ul li a {
+  color: rgb(233, 230, 225);
+}
+
+/* main-menu */
+.main-menu {
+  background: white;
+  height: 15%;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 8%;
+  padding-bottom: 1.8%;
+}
+
+.main-menu img {
+  width: 80px;
+  height: 60px;
+  margin-top: 1.5%;
+}
+
+.main-menu p {
+  color: rgb(40, 94, 40);
+  font-size: 30px;
+  font-weight: 700;
+  line-height: normal;
+  margin-top: -20%;
+  margin-left: 32%;
+  width: 100%;
+}
+
+.main-menu p span {
+  color: rgb(235, 124, 21);
+}
+
+.main-menu-ul {
+  display: flex;
+  flex-direction: row;
+  width: 40%;
+}
+
+.main-menu-ul li {
+  padding-top: 7%;
+  padding-bottom: 2%;
+  padding-right: 7%;
+}
+
+.main-menu-ul li.disapper {
+  opacity: 0;
+}
+
+.main-menu-ul li a {
+  color: rgb(26, 25, 25);
+  margin-right: 15px;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: normal;
+  padding: 10%;
+  transition: all 0.3s linear;
+}
+
+.main-menu-ul li a:hover {
+  border: 2px outset  rgb(235, 124, 21);
+}
+
+.main-menu-ul li a.active {
+  color: rgb(235, 124, 21);
+}
+
+.main-menu .icon {
+  opacity: 0;
+}
+
+/* headline */
+.back img {
+  width: 100%;
+  height: 45vh;
+}
+
+.headline {
+  background: lightgray;
+  height: 135vh;
+}
+
+.headline .explain {
+  margin-left: 10%;
+}
+
+.headline .explain p.p1 {
+  color: rgb(207, 71, 37);
+  margin-top: 15px;
+  font-size: 28px;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.headline .explain h1 {
+  font-weight: 900;
+  max-width: 800px;
+  margin-top: 10px;
+  font-size: 55px;
+  line-height: 1.3;
+  background-image: linear-gradient(-225deg, #ec5242 0%, #44107a 29%, #ec5242 67%, #ec5242 100%);
+  background-clip: border-box;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
```

Comment:
- [ ] Please know that your main title should be using the font `COCOGOOSE` instead of `Lato`:
  ![image](https://user-images.githubusercontent.com/20567503/198011145-03e81346-1cb8-46d3-a2da-a197aab2b173.png)
  Kindly change it to the correct font. 😄